projects/vdk-core/tests/vdk/internal/builtin_plugins/run/test_data_job.py

```diff
@@ -81,22 +84,24 @@ def test_run_not_from_template(mock_store_set):
     ]
 
 
-@patch("vdk.internal.core.statestore.StateStore.set")
-@patch(
-    "vdk.internal.core.statestore.StateStore.get", MagicMock(return_value=[])
-)  # for creating child context
-def test_run_from_template(mock_store_set):
-    template_name = "template_name"
-    job_builder = DataJobBuilder()
-    job_builder.build().run(template_name=template_name)
-    mock_store_set.assert_called_with(StoreKey(key="vdk.template_name"), template_name)
-    assert (
-        len(
-            [
-                c
-                for c in mock_store_set.call_args_list
-                if c[0][0] == StoreKey(key="vdk.template_name")
-            ]
-        )
-        == 1
-    )
+# TODO: Find a way to test this without mocking the state store
```

Comment:
Not sure if we should even do this at all. This test has very little value, in my opinion.